yamscripts.yml

```diff
@@ -65,7 +65,7 @@ scripts:
 
 # lint: Run eslint and sass-lint
   lint:
-    eslint-check: eslint --cache --print-config . | eslint-config-prettier-check
+    eslint-check: eslint --cache --print-config .eslintrc.js | eslint-config-prettier-check
```

Comment:
What's this change about?  I suspect that whatever we do, we want a comment describing why we've chosen the argument we have, if it's not a file that's part of the app, as suggested by the eslint-config-prettier readme.

Reply:
Ok, I missed the original description at the top of the PR, but I suspect we still want to just use a file from the app as suggested by that README, or else comment it.

Reply:
Switched to AboutNewTabService.jsm